community_server/src/Model/Messages/GPBMetadata/Gradido/BasicTypes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: gradido/BasicTypes.proto
+
+namespace GPBMetadata\Gradido;
+
+class BasicTypes
+{
+    public static $is_initialized = false;
+
+    public static function initOnce() {
+        $pool = \Google\Protobuf\Internal\DescriptorPool::getGeneratedPool();
+
+        if (static::$is_initialized == true) {
+          return;
+        }
+        $pool->internalAddGeneratedFile(hex2bin(
+            "0aca030a186772616469646f2f426173696354797065732e70726f746f12" .
+            "0d70726f746f2e6772616469646f22380a034b657912110a076564323535" .
+            "313918022001280c480012170a0d656432353531395f7265663130180320" .
+            "01280c480042050a036b657922580a0d5369676e61747572655061697212" .
+            "0e0a067075624b657918012001280c12110a076564323535313918022001" .
+            "280c480012170a0d656432353531395f726566313018032001280c480042" .
+            "0b0a097369676e6174757265223d0a0c5369676e61747572654d6170122d" .
+            "0a077369675061697218012003280b321c2e70726f746f2e677261646964" .
+            "6f2e5369676e617475726550616972222b0a0954696d657374616d70120f" .
+            "0a077365636f6e6473180120012803120d0a056e616e6f73180220012805" .
+            "22230a1054696d657374616d705365636f6e6473120f0a077365636f6e64" .
+            "7318012001280322300a0e5472616e73666572416d6f756e74120e0a0670" .
+            "75626b657918012001280c120e0a06616d6f756e7418022001281222400a" .
+            "08486564657261494412100a0873686172644e756d18012001280312100a" .
+            "087265616c6d4e756d18022001280312100a08746f7069634e756d180320" .
+            "012803620670726f746f33"
```

Comment:
Can you explain what this is about?

Reply:
This is autogenerated from protobuf compiler. 
It didn't need to be in git repository, it is also possible to call the parse_proto.sh in root-folder of community-server to get this if correct version of protoc is accessable. But now with monorepository I can maybe update the build code to use protoc from login-server to generate this files rather than storing it in git-repository.

I don't know how this work exactly and for what this Metadata files are needed.

Reply:
Ok thanks for the explanation, I will make an issue to investigate the generation on compile.